README.md

```diff
@@ -1 +1,394 @@
-# math-magicians-rep
\ No newline at end of file
+# math-magicians-rep
+"math magicians rep" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:  Make simple calculations and Read a random math-related quote.
```

Comment:
[OPTIONAL] You currently have multiple headers or titles in your readme and this is giving it a less professional look😉. I suggest you have just one title and description😉. You can take this description away since you already have one after the table of content😉.

<img width="236" alt="image" src="https://github.com/fickryiman/math-magicians-rep/assets/64643995/1ffa4304-1328-4dd7-830f-8ec47c144082">